src/index.test.ts

```diff
@@ -130,6 +130,34 @@ it('outputs bnb list correctly', async () => {
   )
 })
 
+it('outputs celo list correctly', async () => {
+  const tokenList = await chainifyTokenList(
+    [ChainId.CELO],
+    sampleL1TokenList_2
+  )
+  const sample = celoedSampleTokenList.tokens.map((t) => [
+    t.address,
+    t.chainId,
+    t.extensions,
+  ])
+  const sampleMap = tokenList?.tokens.map((t) => [t.address, t.chainId, t.extensions])
+  console.log(sampleMap)
+  console.log(sample)
+  expect(tokenList).toBeDefined()
+  expect(tokenList?.version).toEqual(celoedSampleTokenList.version)
+  console.log(tokenList )
```

Comment:
```suggestion
  expect(tokenList).toBeDefined()
  expect(tokenList?.version).toEqual(celoedSampleTokenList.version)
```

removed console logs